### ISSUE TITLE
Forecastie as CyanogenMod weather provider

### ISSUE THREAD

**Issue by sunng87** [martykan/forecastie#147]
I knew there is an official OpenWeatherMap provider but it's never released to PlayStore or FDroid. I wish Forecastie could implement their API and be used as a weather provider.

**Comment by robinpaulson** [comment]
Yep, this would be excellent, getting weather on the lock screen

**Comment by robinpaulson** [comment]
Are there any apps, with source code available, which are cyanogenmod (lineageOS now of course) weather providers?

**Comment by mtbu** [comment]
This is the official one: https://github.com/LineageOS/android_packages_apps_OpenWeatherMapProvider

**Comment by robinpaulson** [comment]
I'm afraid I don't have the time at the moment, but if someone were to submit code and a pull request.....
